src/Entity/Speaker.php

```diff
@@ -66,7 +66,7 @@ public function __construct(
     private $biography;
 
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
```

Comment:
I think we said it should be either the default value or the uploaded path, so no null it is.